recipes/fury/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "fury" %}
+{% set version = "0.3.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: ad0eff6921823b8e261c5e4c7339c549b12f9bc6cc05ab4a51aae899ac067ca6
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+
+  run:
+    - python
+    - numpy
+    - scipy
+    - vtk
+    - matplotlib
+    - six
+
+test:
+  imports:
+    - fury
+
+about:
+  home: http://github.com/fury-gl/fury
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: LICENSE.txt
+  summary: 'Free Unified Rendering for Python - Scientific Visualization'
+  description: |
+    Free Unified Rendering for Python (FURY) is a library that offers
+    a collection of new widgets, functionnality to vtk. It works under
+    Python 2.7 and Python 3.4+
+  doc_url: http://fury.gl/
+  dev_url: https://github.com/fury-gl/fury
+
+extra:
+  recipe-maintainers:
+    - skoudoro
+    - garyfallidis
```

Comment:
@garyfallidis , Could you please confirm that you also like to be a maintainer of this recipe?

Reply:
Yes! Thank you @marcelotrevisani !